### ISSUE TITLE
Installation failure on Mageia

### ISSUE THREAD

**Issue by ggemmill** [MeanEYE/Sunflower#493]
Environment: Mageia 7
Downloaded the sunflower 0.4 .62-2 rpm but installation fails:
A requested package cannot be installed:
sunflower-0.4.62-2.noarch (due to unsatisfied gtk3)
But 
# urpmi gtk+3.0
Package gtk+3.0-3.24.8-1.mga7.x86_64 is already installed
and 23 packages were installed with 
# urpmi gtk+3.0-devel
Any ideas to complete installation?
TIA
graeme

**Comment by MeanEYE** [comment]
Hm,

Sunflower requires gir packages for GTK3 as well. They are GObject Introspection Resources, which allow us to use native
GTK library from Python. 

Can you search for gir packages and see if they exist. It's most likely us requiring packages that don't exist.

-- 
Mladen Mijatov
Head of R&D, Way2CU
CEO, AGM Development

Key ID: 4096R/83EFD5A0 2013-08-18

**Comment by ggemmill** [comment]
# urpmi gobject-introspection
Package gobject-introspection-1.60.1-2.mga7.x86_64 is already installed
Is this a version problem?

**Comment by MeanEYE** [comment]
I don't think it is. It's most likely my poor knowledge of Mageia and its package names. I'll check and see if I can work something out.
